pkgs/top-level/python-packages.nix

```diff
@@ -14387,7 +14387,11 @@ self: super: with self; {
 
   soupsieve = callPackage ../development/python-modules/soupsieve { };
 
-  sourmash = callPackage ../development/python-modules/sourmash { };
+  sourmash = callPackage ../development/python-modules/sourmash {
+    stdenv = if pkgs.stdenv.isDarwin then pkgs.overrideSDK pkgs.stdenv {
+      darwinMinVersion = "10.14";
+    } else pkgs.stdenv;
```

Comment:
```suggestion
    stdenv = if stdenv.isDarwin then overrideSDK stdenv {
      darwinMinVersion = "10.14";
    } else stdenv;
```

Reply:
Moved back into package, followed the example from [renovate](https://github.com/NixOS/nixpkgs/blob/1e4d24952a0671a849ad5d61bcead24e2c398458/pkgs/by-name/re/renovate/package.nix#L17-L21)